src/main/scala/strawman/collection/mutable/Seq.scala

```diff
@@ -0,0 +1,76 @@
+package strawman
+package collection
+package mutable
+
+trait Seq[A] extends strawman.collection.Seq[A] with Growable[A] {
```

Comment:
Ah, cool. I completely misremembered what `mutable.Seq` was so far. I thought it had `+=` as well as `update`. So, yes, indeed, we can have `GrowableSeq` underneath `Seq`.